src/antworld/render_mesh.cc

```diff
@@ -15,22 +15,30 @@ namespace BoBRobotics
 {
 namespace AntWorld
 {
-RenderMesh::RenderMesh() : m_VAO(0), m_PositionVBO(0), m_TextureCoordsVBO(0), m_IBO(0), m_NumIndices(0)
+void RenderMesh::render() const
 {
+    // Bind surface, leaving texture binding alone
+    m_Surface.bind(false);
+
+    // Render surface
+    m_Surface.render(GL_QUADS);
+
+    // Unbind surface, leaving texture binding alone
+    m_Surface.unbind(false);
 }
+
+//----------------------------------------------------------------------------
+// BoBRobotics::AntWorld::RenderMeshCylinder
 //----------------------------------------------------------------------------
```

Comment:
Maybe a one-line comment would be good here just to explain what ``RenderMeshCylinder`` is and how it differs from ``RenderMesh``.

I'm well aware that being a bit of a hypocrite here :wink:

Reply:
So, good thing you raised this, turns out it's not a cylinder in any way so renamed the class and added a brief explanation of what it does